id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/validator/BaseIdRepoValidator.java

```diff
@@ -62,18 +62,27 @@ protected void validateReqTime(LocalDateTime reqTime, Errors errors) {
 			errors.rejectValue(REQUEST_TIME, MISSING_INPUT_PARAMETER.getErrorCode(),
 					String.format(MISSING_INPUT_PARAMETER.getErrorMessage(), REQUEST_TIME));
 		} else {
-			if (DateUtils.after(reqTime, DateUtils.getUTCCurrentDateTime()
-					.plusMinutes(EnvUtil.getDateTimeAdjustment()))) {
+			LocalDateTime currentUtcTime = DateUtils.getUTCCurrentDateTime();
+			LocalDateTime adjustedCurrentUtcTime = currentUtcTime.plusSeconds(EnvUtil.getDateTimeAdjustment());
+
+			int maxDeviationSeconds = 60;

```

Comment:
Add a property `idrepo-max-request-time-deviation-seconds` for taking this value. Keep 60 as default